Tests/Invoke-Changes.Tests.ps1

```
@@ -0,0 +1,351 @@
+BeforeAll {
+    function Import-RegistryFile { param($Message, $RegistryFile) }
```

PSScriptAnalyzer (1.25.0)

[warning] The parameter 'Message' has been declared but not used.

(PSReviewUnusedParameter)